regions/shapes/circle.py

```diff
@@ -43,8 +44,16 @@ def to_shapely(self):
         return self.center.to_shapely().buffer(self.radius)
 
     def to_sky(self, mywcs, mode='local', tolerance=None):
```

Comment:
`mywcs` is a weird argument name that's not used in other Astropy or affiliated package codes as far as I know.
Change to `wcs` possible here, no?